pkg/specgen/generate/container_create.go

```diff
@@ -153,7 +153,17 @@ func MakeContainer(ctx context.Context, rt *libpod.Runtime, s *specgen.SpecGener
 	if err != nil {
 		return nil, err
 	}
-	return rt.NewContainer(ctx, runtimeSpec, options...)
+
+	ctr, err := rt.NewContainer(ctx, runtimeSpec, options...)
+	if err != nil {
+		return ctr, err
+	}
+
+	// Copy the content from the underlying image into the newly created
+	// volume if configured to do so.
+	verr := rt.PrepareVolumeOnCreateContainer(ctx, ctr)
```

Comment:
Jusy use err.
I don't see the data being copied up in PrepareVolumeOnCreateContainer?

Reply:
Using `err` was compilation error. I had to change that.
`PrepareVolumeOnCreateContainer()` calls `mountStorage()` that internally calls `mountNamedVolume()` which does the copy.

Reply:
This should work.
```
err = rt.PrepareVolumeOnCreateContainer(ctx, ctr)
```

Reply:
I changed the code to not worry about storing in variable. Please take another look.